src/generator/AutoRest.Go/Templates/VersionTemplate.cshtml

```diff
@@ -24,8 +24,8 @@ import (
 @EmptyLine
 
 const (
-    major = "3"
-    minor = "1"
+    major = "4"
```

Comment:
If there is an option for PackageVersion, I will add that in rakefile(which we use to generate SDK) in coming PR.